### DISCUSSION TITLE
Is it possible with File Centipede to download all of the videos on a specific YouTube channel on its highest quality?

### DISCUSSION THREAD

**Discussion by Shnxxx** [filecxx/FileCentipede#618] [Q&A]
Hi! As the title suggests, is it possible to save an offline copy of an entire YouTube channel? I would like to have a local copy of the channel's content for traveling purposes, enabling me to watch it without relying on a signal or internet connection.

Thank you!!!

**Comment by filecxx** [comment]
FileCentipede does not currently support downloading entire YouTube channels. However, there are several other software options available that can accomplish this task.
